### ISSUE TITLE
Some Programs Get Absorbed

### ISSUE THREAD

**Issue by ibrokemypie** [lbonn/i3-quickterm#2]
Certain programs such as Discord and Lutris seem to get absorbed, becoming the dropdown instead of the terminal...

**Comment by lbonn** [comment]
Yes, It seems to happen sometimes, another window must have taken the focus between the start of the program and the i3 `mark` command (add a mark to the currently focused window).

Does it occur 100% of the time?
I'm not familiar with these applications, do they run "always on top" when used with a floating WM?

**Comment by ibrokemypie** [comment]
Yes it does happen 100% of the time and neither are always on top. It seems to happen across workspaces too.

**Comment by ibrokemypie** [comment]
Update, has started happening to chrome as well
